SofaKernel/SofaFramework/config.h.in

```diff
@@ -22,6 +22,11 @@
 #ifndef SOFA_CONFIG_H
 #define SOFA_CONFIG_H
 
+#include <sofa/config/sharedlibrary_defines.h>
+#include <sofa/config/build_option_dump_visitor.h>
+#include <sofa/config/build_option_opengl.h>
+#include <sofa/config/build_option_threading.h>
```

Comment:
Do you think we should deprecate this? When?

Reply:
I don't know. But would prefer to focus of task with more added value :)